src/libPMacc/include/memory/buffers/Buffer.hpp

```diff
@@ -47,12 +47,13 @@ namespace PMacc
 
         typedef DataBox<PitchedBox<TYPE, DIM> > DataBoxType;
 
-        /**
-         * constructor
-         * @param dataSpace description of spread of any dimension
+        /*
+         * @param dataSpace description of spread of any dimension (in elements)
+         *                  can be lesser than `physicalMemorySize`
```

Comment:
it would be great if the contructor documentation would write *why* `physicalMemorySize` might be larger than `dataSpace` (which is badly named):

one example would be a buffer that exists as a view of an other buffer, isn't it?